knx/dpt/types_registry.go

```diff
@@ -81,36 +75,30 @@ func (r *Registry) Init() error {
 
 	// Register the types
 	for _, d := range dpts {
-		err := r.add(d)
+		err = r.add(d)
 		if err != nil {
-			return err
+			return
 		}
 	}
 
-	return nil
+	return
 }
 
 func (r Registry) List() []string {
 	// Initialize the key-list
-	keys := make([]string, len(r.lut))
+	keys := make([]string, len(r))
 
 	// Fill the key-list
 	i := 0
-	for k := range r.lut {
+	for k := range r {
 		keys[i] = k
 		i++
 	}
 
 	return keys
 }
 
-func (r Registry) Lookup(name string) (t DatapointValue, err error) {
-	t, ok := r.lut[name]
-	if ok {
-		err = nil
-	} else {
-		err = fmt.Errorf("datapoint-type \"%v\" not found!", name)
-	}
-
+func (r Registry) Lookup(name string) (d DatapointValue, ok bool) {
```

Comment:
I think you inherit the `[]` operator. Therefore this method is redundant.

Reply:
Just thought it might be a nice-to-read function call but I'll remove it if you don't like it.

Reply:
Fair enough, makes sense to keep it.